sys-apps/shadow/shadow-4.4.ebuild

```diff
@@ -117,15 +131,14 @@ src_install() {
 	newins etc/login.defs login.defs
 	dosym ../usr/share/shadow/login.defs /etc/login.defs
 
+	set_login_opt CREATE_HOME yes
```

Comment:
If it is a behavior we want it would need to be in PAM since login is only used by getty and not SSH.